src/main/java/ru/urururu/sanity/api/cfg/CfePrinter.java

```diff
@@ -153,6 +153,7 @@ public void visit(Switch switchElement) {
             print(entry.getKey());
             sb.append(" -> ").append(getId(entry.getValue()));
         }
+        sb.setLength(sb.length() - 1); // todo get rid of trailing whitespace in getId
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)